.ci/docker/build.sh

```diff
@@ -218,6 +220,15 @@ case "$image" in
     CONDA_CMAKE=yes
     TRITON=yes
     ;;
+  pytorch-linux-jammy-xpu-2024.0-py3)
+    ANACONDA_PYTHON_VERSION=3.8
+    GCC_VERSION=11
+    PROTOBUF=yes
+    DB=yes
+    VISION=yes
+    BASEKIT_VERSION=2024.0.0-49522
+    CONDA_CMAKE=yes
```

Comment:
Compared to other backends, may I know why we do not need to define `NINJA_VERSION` and `TRITON`?

Reply:
Add NINJA_VERSION back, but for TRITON still need wait for the triton xpu backend update to latest version.

Reply:
So, could you please add TODO?

Reply:
Added